src/sc/embedder/content/embedder.py

```diff
@@ -18,9 +17,10 @@
 from plone.namedfile.file import NamedImage as ImageValueType
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from sc.embedder import MessageFactory as _
+from sc.embedder.interfaces import IConsumer
 from z3c.form import button
-from z3c.form.interfaces import IFormLayer
 from z3c.form.interfaces import IFieldWidget
+from z3c.form.interfaces import IFormLayer
 from z3c.form.widget import FieldWidget
 from zope import schema, component
 from zope.component import adapter
```

Comment:
How about to follow plone.api conventions here http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#grouping-and-sorting ?